### PR TITLE
Create Database Settings page.

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

It creates a Database settings page. The Page shows details about a particular database and buttons to enable resetting and deleting a database.

#### Ticket

https://trello.com/c/O4TuxuT8

#### Screenshots
**NOTE** Watch below.

captured (4)

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -65,6 +66,7 @@ const Routes = () => (
       <Route path="/verify/:token" component={VerificationSentPage} />
       <Route path="/reset_password/:token" component={CreateNewPassword} />
       <Route path="/team" component={TeamPage} />
+      <Route path="/database" component={DBSettingsPage} />
```
Comment (Mubangizi) [inline src/router.js:69]: Is this the best route you can put the settings page? just inquiring

Comment (neelxie) [reply src/router.js:69]: @Mubangizi given that the endpoint hasn't been made and all that is dummy data I figured that all would change when plugging in.

Comment (colinwagaba) [reply src/router.js:69]: ideally it should under `/databases/:dbID/[maybe settings]`

**Review by colinwagaba — APPROVED** [review]
can't wait to see how it plays out with live data...

**Pull request merged**